macros/contexts/contextLinearRelation.pl

```diff
@@ -0,0 +1,399 @@
+the variables if your situation should use something different. For example with
+
+    Context()->variables->are(m => 'Real', n =>'Real');
+
+There is one special context flag.
+
+=over 
+
+=item S<C<< standardForm >>>
+
+This determines whether something like C<<LinearRelation("x+2 < y+z")>> will be
+displayed as C<<x+2 < y+z>> or converted to standard form: C<<x-y-z < -2>>.
+It is 0 by default.
+
+=back 
+
+There is one special method for LinearRelation objects.
+
+=over
+
+=item S<C<< $LR->check_at(point) >>>
+
+This returns true or false depending on if the point satisfies the relation.
+C<point> must be a MathObject Point, Vector, or ColumnVector; or simply be an
+array reference. The number of entries in C<point> must match the number of
+variables in the context.
+
+=back
+
+=cut
+
+loadMacros('MathObjects.pl');
+
+sub _contextLinearRelation_init { LinearRelation::Init() };    # don't reload this file
+
+##################################################
+#
+#  Initialize the contexts and make the creator function.
+#
+
+package LinearRelation;
+our @ISA = qw(Value::Formula);
+
+sub Init {
+	my $context = $main::context{LinearRelation} = Parser::Context->getCopy("Vector");
+	$context->{name}                       = "LinearRelation";
+	$context->{precedence}{LinearRelation} = $context->{precedence}{special};
+	$context->{value}{LinearRelation}      = "LinearRelation";
+	$context->{value}{Formula}             = "LinearRelation::formula";
+	$context->parens->remove("<");
+	$context->flags->set(standardForm => 0);
+	$context->operators->add(
+		'=' => {
+			precedence    => .5,
+			associativity => 'left',
+			type          => 'bin',
+			string        => ' = ',
+			kind          => 'eq',
+			class         => 'LinearRelation::inequality',
+			formulaClass  => "LinearRelation"
+		},
+
+		'<' => {
+			precedence    => .5,
+			associativity => 'left',
+			type          => 'bin',
+			string        => ' < ',
+			kind          => 'lt',
+			class         => 'LinearRelation::inequality',
+			formulaClass  => "LinearRelation"
+		},
+		'>' => {
+			precedence    => .5,
+			associativity => 'left',
+			type          => 'bin',
+			string        => ' > ',
+			kind          => 'gt',
+			reverse       => 'lt',
+			class         => 'LinearRelation::inequality',
+			formulaClass  => "LinearRelation"
+		},
+
+		'<=' => {
+			precedence    => .5,
+			associativity => 'left',
+			type          => 'bin',
+			string        => ' <= ',
+			TeX           => '\le ',
+			kind          => "le",
+			class         => 'LinearRelation::inequality',
+			formulaClass  => "LinearRelation",
+			alternatives  => ["\x{2264}"]
+		},
+		'=<' => {
+			precedence    => .5,
+			associativity => 'left',
+			type          => 'bin',
+			string        => ' <= ',
+			TeX           => '\le ',
+			kind          => "le",
+			class         => 'LinearRelation::inequality',
+			formulaClass  => "LinearRelation"
+		},
```

Comment:
You can just use

``` perl
                '=<' => {
                         alias => '<='
		},
```

here (that wasn't available when the Inequalities content was created).

Reply:
Does it make sense to handle `≤` etc this way too? With `alias` instead of with `alternatives`?

Should both `=<` and `≤` be `alternatives`? Then they only work if `parseAlternatives` is set.

Reply:
> Does it make sense to handle `≤` etc this way too?

That would work, but would mean it was always defined, not just when `parseAlternatives` is set.  That would make this inconsistent with other alternative notations.

> Should both =< and ≤ be alternatives? 

That would also be possible.  It depends on whether you want that to be based on `parseAlternatives` or not.  I don't feel strongly either way, with perhaps a slight lean toward both as alternatives.